modules/express/src/retryPromise.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Thrown in `retryPromise()`
+ */
+export class ErrorMaxTries extends Error {
+  constructor(tries: number) {
+    super(`giving up after ${tries} tries`);
```

Comment:
when extending `Error` you must set the prototype otherwise it will break the prototype chain
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-newtarget

Reply:
Thanks for the feedback @lifeiscontent! Is this just a matter of adding the following line under `super()`?
```
Object.setPrototypeOf(this, new.target.prototype);
```

Reply:
@lifeiscontent that fix is no longer needed since we switched to targeting ES2019